. huskyrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this file necessary? I don't see this referenced in the lint-staged documentation at all. In fact, it looks like the latest version of lint-staged dropped husky in favor of simple-git-hooks.